devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy

```diff
@@ -223,3 +240,45 @@ private getModelBindings(model) {
 	}
 	bindings
 }
+
+private Map parseAduroSmartButtonMessage(Map descMap){
+	def buttonState = "pushed"
+	def buttonNumber = 0
+	if (descMap.clusterInt == zigbee.ONOFF_CLUSTER) {
+		if (descMap.command == "01") {
+		    buttonNumber = 1
+		}else if (descMap.command == "00") {
```

Comment:
nit: spaces before `else if` in this function too

Reply:
Fixed issues